### DISCUSSION TITLE
passing info from fastapi to embedded idom

### DISCUSSION THREAD

**Discussion by acivitillo** [idom-team/idom#709] [Question] [answered]
idom is very easy to embed into a FastAPI app, that part is great. I am wondering if I can send FastAPI serialized data to an idom app? One use case for this is users logging into FastAPI and then sending the user info into the IDOM embedded apps.

**Comment by rmorshea** [comment, marked as answer]
That should be addressed by: https://github.com/idom-team/idom/issues/669

I'll make implementing this my current focus.
